【Bug Report】用yarn安装，安装过程中报错，某yarnpkg.com页面反馈404

用yarn安装，安装过程中报错
```
yarn install v1.22.4
[1/4] Resolving packages...
[2/4] Fetching packages...

error An unexpected error occurred: "https://registry.yarnpkg.com/@koa/router/-/router-9.0.0.tgz: Request failed \"404 Not Found\"".
```
[https://registry.yarnpkg.com/@koa/router/-/router-9.0.0.tgz](https://registry.yarnpkg.com/@koa/router/-/router-9.0.0.tgz) 这个打不开。
### version
```
│ revision control │ git                                                                         │
│ remote url       │ https://github.com/DIYgod/RSSHub.git                                        │
│ repository root  │ /var/www/RSSHub                                                             │
│ last update      │ 2020-04-19T13:09:40.434Z                                                    │
│ revision         │ 20fc36affc54dfda4d109141c482f361a3ca24d0                                    │
│ comment          │ Merge pull request #4456 from DIYgod/dependabot/npm_and_yarn/vuepress-1.4.1 │
│ branch           │ master 
```

```
┌──────────┬────┬─────────┬──────┬───────┬─────────┬─────────┬────────┬─────┬────────┬──────┬──────────┐
│ App name │ id │ version │ mode │ pid   │ status  │ restart │ uptime │ cpu │ mem    │ user │ watching │
├──────────┼────┼─────────┼──────┼───────┼─────────┼─────────┼────────┼─────┼────────┼──────┼──────────┤
│ rsshub   │ 0  │ 0.1.1   │ fork │ 10066 │ stopped │ 31      │ 0      │ 0%  │ 0 B    │ root │ disabled │
└──────────┴────┴─────────┴──────┴───────┴─────────┴─────────┴────────┴─────┴────────┴──────┴──────────┘
 Use `pm2 show <id|name>` to get more details about an app
```

### 谢谢

用 npm install 报错如下
```
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @koa/router@9.0.0
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
```

--- a/package.json
+++ b/package.json
@@ -59,7 +59,7 @@
     "yorkie": "2.0.0"
   },
   "dependencies": {
-    "@koa/router": "9.0.0",
+    "@koa/router": "8.0.8",
     "@postlight/mercury-parser": "2.2.0",
     "@sentry/node": "5.15.4",
     "aes-js": "3.1.2",

